plugins/experimental/url_sig/url_sig.c

```diff
@@ -570,7 +606,7 @@ TSRemapDoRemap(void *ih, TSHttpTxn txnp, TSRemapRequestInfo *rri)
     rri->redirect = 1;
     break;
   default:
-    TSHttpTxnErrorBodySet(txnp, TSstrdup("Authorization Denied"), strlen("Authorization Denied") - 1, TSstrdup("text/plain"));
+    TSHttpTxnErrorBodySet(txnp, TSstrdup("Authorization Denied"), sizeof("Authorization Denied") - 1, TSstrdup("text/plain"));
```

Comment:
Why use `sizeof` here and `strlen` on `EXP_QSTRING` and `ALG_QSTRING` earlier?

Reply:
Because I'm not a good person.